### PR TITLE
refactor(i18n): camelcased keys

### LINKED ISSUE #1217: request: make all i18n keys camelCase
**Is your feature request related to a problem? Please describe.**

Klasa's convention is to have all keys upper-case, however in i18next, the convention is to have them camelCase (with the occasional _snake for plural rules).

**Describe the solution you'd like**

Change all keys from SCREAMING_SNAKE_CASE to camelCase

**Describe alternatives you've considered**

1. Leaving it be? Feels weird to not follow the language standards.
1. Do this while we switch? Might increase development cost by a lot.

**Additional context**

N/a.

### PR DESCRIPTION
**Pull request by favna**
closes #1217

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,8 +44,8 @@
 		"url": "https://github.com/skyra-project/skyra/issues"
 	},
 	"dependencies": {
-		"@influxdata/influxdb-client-apis": "^1.6.0",
 		"@influxdata/influxdb-client": "^1.6.0",
```
Comment (favna) [inline package.json:47]: @QuantumlyTangled I told ya Yarn would put it back eventually lmfao

**Review by Quantumlyy — DISMISSED** [review]

**Review by Rexogamer — DISMISSED** [review]
this was a big one

**Review by Quantumlyy — COMMENTED** [review]
Are the changes to transform Enum member names to camelCase on purpose or a side effect of the script?

**Review by cfanoulis — DISMISSED** [review]
not a thorough review but LGTM

**Comment by favna** [comment]
@QuantumlyTangled which files? It's probably unintentional but I must've looked over them if it's still the case.

**Review by Quantumlyy — DISMISSED** [review]

**Review by kyranet — APPROVED** [review]

**Review by Quantumlyy — APPROVED** [review]

**Review by Rexogamer — APPROVED** [review]

**Pull request merged**